File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1094.lua
n = io.read("*number")
s=0
r=0
c=0

for i=1,n,1 do
    v = io.read("*number")
    animal = io.read()

    if(animal==' C') then
        c=c+v;
    elseif(animal==' R') then
        r=r+v;
    else
        s=s+v;
    end
end
soma = c+r+s

print(string.format("Total: %d cobaias", c+r+s));
print(string.format("Total de coelhos: %d", c));
print(string.format("Total de ratos: %d", r));
print(string.format("Total de sapos: %d", s));
print(string.format("Percentual de coelhos: %.2f %%", (100*c)/soma));
print(string.format("Percentual de ratos: %.2f %%", (100*r)/soma));
print(string.format("Percentual de sapos: %.2f %%", (100*s)/soma));

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1094.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
    
        int n = ler.nextInt(), v, s=0, r=0, c=0, i;
        char animal;

        for(i=0;i<n;i++){
            v = ler.nextInt();
            animal = ler.next().charAt(0);
            if(animal=='C'){
                c+=v;
            } else if(animal=='R'){
                r+=v;
            } else{
                s+=v;
            }
        }
        double soma = c+r+s;
    
        System.out.printf("Total: %d cobaias\n", c+r+s);
        System.out.printf("Total de coelhos: %d\n", c);
        System.out.printf("Total de ratos: %d\n", r);
        System.out.printf("Total de sapos: %d\n", s);
        System.out.printf("Percentual de coelhos: %.2f %%\n", (100*c)/soma);
        System.out.printf("Percentual de ratos: %.2f %%\n", (100*r)/soma);
        System.out.printf("Percentual de sapos: %.2f %%\n", (100*s)/soma);
        
    }
}